Lib/test/test_pprint.py

```diff
@@ -1123,6 +1124,217 @@ def test_user_string(self):
     'jumped over a '
     'lazy dog'}""")
 
+    def test_block_style(self):
+        @dataclasses.dataclass
+        class DummyDataclass:
+            foo: str
+            bar: float
+            baz: bool
+            qux: dict = dataclasses.field(default_factory=dict)
+            quux: list = dataclasses.field(default_factory=list)
+            corge: int = 1
+            garply: tuple = (1, 2, 3, 4)
+
+        dummy_dataclass = DummyDataclass(foo="foo", bar=1.2, baz=False,
+                                         qux={"foo": "bar", "baz": 123},
+                                         quux=["foo", "bar", "baz"], corge=7,
+                                         garply=(1, 2, 3, 4))
+        dummy_dict = {"foo": "bar", "baz": 123,
+                      "qux": {"foo": "bar", "baz": 123},
+                      "quux": ["foo", "bar", "baz"], "corge": 7 }
+        dummy_ordered_dict = collections.OrderedDict([("foo", 1), ("bar", 12),
+                                                      ("baz", 123)])
+        dummy_list = ["foo", "bar", "baz"]
+        dummy_tuple = ("foo", "bar", "baz", 4, 5, 6, dummy_list)
+        dummy_set = { 1, 2, 3 }
```

Comment:
my bad, revert next change
```suggestion
        dummy_set = { (1, 2, 3) }
```